src/integration/InterfaceUtils.cpp

```diff
@@ -80,7 +93,7 @@ InterfaceUtils::applyModality(const std::unordered_map<std::string, boost::any>
 
 	// Retrieve epistemicOperator and check if it is "BELIEF"
 	auto epistemicOperator = boost::any_cast<int>(options.at("epistemicOperator"));
-	if (epistemicOperator == 1) {
+	if (epistemicOperator == EpistemicOperator::BELIEF) {
```

Comment:
I am a bit confused about the whole "applyModalOperator" thing. Maybe you can explain to me offline?

Reply:
As discussed: This is necessary for ROS or other interfaces that provide modalities in external map like structures